### ISSUE TITLE
sdl: slow window resize

### ISSUE THREAD

**Issue by stsp** [dosemu2/dosemu2#1572]
When TTF was implemented,
window resize was very smooth.
But now its jerky and slow.

**Comment by stsp** [comment]
Bisect: 236794f25

**Comment by stsp** [comment]
Such a horrible regression stayed
for half an year. :(
@stefand would you like to re-test
that? The original patch says you
tested it. Can't remember what was
the deal exactly though.

**Comment by stefand** [comment]
My life got pretty busy lately, I don't know when I'll find time to spend time debugging this. I remember I also promised you some action on the threading pull request :-(

**Comment by stsp** [comment]
I am not sure any debugging
is needed here. I just think commit
236794f helped with some hw-accel
rendering, which is what you tested.
So I suppose the fix of the aforementioned
commit, can get the same testing.

**Comment by stsp** [comment]
That is: fix is already applied.
No debugging is needed.
I just think you had a particular
testing scenario which motivated
the commit that caused the regression
(which is now fixed).

**Comment by stefand** [comment]
I finally got around to updating dosemu2 (and fdpp). I don't see any negative impact of this patch (or the update in general) on my setup. It was only very superficial testing though.

It's my usual setup with THREADED_REND off, hwrend on. Radeon GPU for now (so threaded_rend would work with hwrend, by luck, as discussed a year ago). Graphics and text windows resize fine.

I benchmarked winbench on windows with upstream dosemu2 and e9e7046b64745c23782f16cd50e3385e9a57308e reverted. I got approx 52M points with and without the patch. I don't have a text mode drawing benchmark :-)
